static/app/constants/index.tsx

```diff
@@ -318,6 +326,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Cron Monitors'),
     uid: 13,
   },
+  [DataCategoryExact.PROFILE_DURATION]: {
+    name: DataCategoryExact.PROFILE_DURATION,
+    apiName: 'profileDuration',
+    plural: 'profile durations',
+    displayName: 'profile duration',
+    titleName: t('Profile Durations'),
```

Comment:
i think this should be profile seconds based on https://github.com/getsentry/getsentry/pull/13570